google/cloud/security/common/gcp_api/admin_directory.py

```diff
@@ -0,0 +1,68 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Wrapper for Admin Directory  API client."""
+
+from googleapiclient.errors import HttpError
+from httplib2 import HttpLib2Error
+from ratelimiter import RateLimiter
+
+from google.cloud.security.common.gcp_api._base_client import _BaseClient
+from google.cloud.security.common.gcp_api._base_client import ApiExecutionError
+from google.cloud.security.common.util.log_util import LogUtil
+
+
+LOGGER = LogUtil.setup_logging(__name__)
+
+
+class AdminDirectoryClient(_BaseClient):
+    """GSuite Admin Directory API Client."""
+
+    API_NAME = 'admin'
+    DEFAULT_MAX_QUERIES = 400
```

Comment:
I noticed in the IAM & Admin -> Quotas dashboard, under "All Quotas", that there doesn't seem to be a ratelimit for the Admin SDK, but just the total number of queries that can be made to it in a day. Do you happen to know what would happen if someone should exceed that quota? It looks like the default is 150,000 queries per day.